### PR TITLE
Add `commentstring` support in ftdetect

### PR DESCRIPTION
**Pull request by mkatychev**
* combined `autocmd` statements into a single glob pattern

### CODE REVIEW

**Comment by tgross35** [comment]
Does adding either `\C` or `(?i)` to the beginning of the string make this match case insensitive? The pattern currently will not match `JUSTFILE` and others, [this section](https://github.com/casey/just#quick-start) says it matches case insensitive.

Not sure how to do insensitive regex in Vim though...

File: ftdetect/just.vim [hunk 1]
```diff
@@ -1,5 +1 @@
-au VimEnter,BufWinEnter,BufRead,BufNewFile *.just setlocal filetype=just
-au VimEnter,BufWinEnter,BufRead,BufNewFile justfile setlocal filetype=just
-au VimEnter,BufWinEnter,BufRead,BufNewFile Justfile setlocal filetype=just
-au VimEnter,BufWinEnter,BufRead,BufNewFile .justfile setlocal filetype=just
-au VimEnter,BufWinEnter,BufRead,BufNewFile .Justfile setlocal filetype=just
+au VimEnter,BufWinEnter,BufRead,BufNewFile {.,}[Jj]ustfile,*.just setlocal filetype=just | setlocal commentstring=#\ %s
```
Comment (tgross35) [inline ftdetect/just.vim:1]:
```suggestion
au VimEnter,BufWinEnter,BufRead,BufNewFile {.,}justfile\c,*.just setlocal filetype=just | setlocal commentstring=#\ %s
```

It seems like `\c` does work

**Comment by tgross35** [comment]
Thanks!

File: ftdetect/just.vim (hunk 1 above)
Comment (mkatychev) [reply ftdetect/just.vim]: Nice catch, apologies for not following up. Somehow these comments didn't land in my inbox

Comment (tgross35) [reply ftdetect/just.vim]: No worries, thanks for the patch 👍

**Pull request merged**